config-templates/ibm/ibm-object-storage-plugin/changelog.json

```diff
@@ -3,6 +3,17 @@
         "2.2": {
             "status": "supported",
             "latest-revision": {
+                "revision": "14",
```

Comment:
Need to update revision/chart version in the template `ibm-object-storage-plugin/2.2/deployment.yaml`

Reply:
Fixed it.